src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_RemoveWorld.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * This file is a part of project QuickShop, the name is SubCommand_RemoveWorld.java
+ *  Copyright (C) PotatoCraft Studio and contributors
+ *
+ *  This program is free software: you can redistribute it and/or modify it
+ *  under the terms of the GNU Lesser General Public License as published by the
+ *  Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT
+ *  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ *  FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ *  for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.maxgamer.quickshop.command.subcommand;
+
+import lombok.AllArgsConstructor;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+import org.maxgamer.quickshop.QuickShop;
+import org.maxgamer.quickshop.command.CommandProcesser;
+import org.maxgamer.quickshop.shop.Shop;
+import org.maxgamer.quickshop.util.MsgUtil;
+import org.maxgamer.quickshop.util.Util;
+
+@AllArgsConstructor
+public class SubCommand_RemoveWorld implements CommandProcesser {
```

Comment:
Codacy found an issue: [The class name 'SubCommand_RemoveWorld' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/Ghost-chu/QuickShop-Reremake/pullRequest?prid=6383074)